xbmc/addons/AddonStatusHandler.h

```diff
@@ -30,7 +30,7 @@ namespace ADDON
       CAddonStatusHandler(const std::string& addonID,
                           uint32_t instanceID,
                           ADDON_STATUS status,
-                          std::string message,
+                          const std::string& message,
```

Comment:
> Was requested by @ksooo

 Really? Ooops.

Reply:
I'm just not a big fan of the "copy and move idiom". This can be discussed back and forth. Do it as you like, but either copy & move or const ref & no move.

Reply:
About the `const std::string& message`, maybe it makes sense to remove complete?
As reason it nowhere becomes set.

Reply:
In another request becomes the unused string removed.